File: exercism/go/archive/two-fer/two_fer.go
// Package twofer prints out the saying
package twofer

// ShareWith receives a name value and outputs a string with that name
func ShareWith(in string) string {
	if len(in) < 1 {
		return "One for you, one for me."
	}

	//	return fmt.Sprintf("One for %s, one for me.", in)
	return "One for " + in + ", one for me."
}

File: exercism/python/two-fer/two_fer.py
"""Module to run the two-fer challenge."""


def two_fer(name: str = "you") -> str:
    """Process the given `name` into a phrase for the two-fer challange.

    :param name: str - provided name for the cookie
    :return: str - return the two-fer statement given the `name`

    Function to receive a name and provide a two-fer statement using that `name`.
    """

    return f"One for {name}, one for me."
